queries.js

```diff
@@ -7,7 +7,7 @@ module.exports.GetEnterpriseUserAccount = gql`
         enterprise(slug: $slug) {
             name
             id
-            members(query: $query, first: 1, role: MEMBER) {
+            members(query: $query, first: 1) {
                 totalCount
                 nodes {
                     type: __typename
```

Comment:
do you still wanna keep `... on EnterpriseUserAccount` below since it needs "admin:enterprise" scope

Reply:
Without the `admin:enterprise` scope we won't be able to get any node information, we need to change the query